CSVTransformStream.js

```diff
@@ -0,0 +1,56 @@
+import { Stream } from 'stream';
+import fs from 'fs';
+
+export class CSVTransformStream extends Stream.Transform {
+  constructor(sourceFile, separator) {
```

Comment:
Could we get rid of `sourceFile` parameter here in order to keep this class more clean?

Reply:
Yes, we could, just do setting coma to the first place in changedChunk.